MovieNight/GenreViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  GenreViewController.swift
+//  MovieNight
+//
+//  Created by Tassia Serrao on 09/07/2017.
+//  Copyright © 2017 Tassia Serrao. All rights reserved.
+//
+
+import UIKit
+
+class GenreViewController: UIViewController , UITableViewDelegate, UITableViewDataSource {
+    
+    @IBOutlet weak var genreTableView: UITableView!
+    var genres = [Genre]()
+    let apiClient = MovieNightApiClient()
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        genreTableView.delegate = self
+        genreTableView.dataSource = self
+
+        loadData()
+    }
+    
+    func loadData() {
+        fetchForGenres()
+    }
+    
+    func fetchForGenres() {
+        apiClient.fetchForGenre { (result) in
+            switch result {
+            case .failure(let error) : print(error)
+            case.success(let resource, _) :
```

Comment:
There an empty space after `case` in the failure line and there isn't one in the success line. Can you make sure they are equal whatever the right way is?